src/lib/api/org/index.ts

```diff
@@ -275,3 +277,108 @@ function convertToSnykProject(projectData: v3ProjectData[]) : SnykProject[] {
   return projects
 }
 
+export async function listTargets(
+  requestManager: requestsManager,
+  orgId: string,
+  config?: {
+    limit?: number,
+    isEmpty?: boolean,
+    origin?: string,
+  }
+): Promise<TargetsResponse> {
+  getApiToken();
+  getSnykHost();
+  debug(`Listing all targets for org: ${orgId}`);
+
+  if (!orgId) {
```

Comment:
if the "orgId" is mandatory to pass to the function, wouldn't it always be there? shouldn't we check if the string in empty instead?

Reply:
we are checking if it's undefined or null here. You can pass undefined in this function what you can't do is not have the parameter. I can also check the length.